packages/components/Atoms/Global/scss/settings/_variables.scss

```diff
@@ -83,3 +83,28 @@ $rpl-dropshadow-color: rgba(0, 0, 0, .14) !default;
 
 // Focus
 $rpl-active-focus-outline-dark: 1px solid rpl-color('white') !default;
+
+// Menu
+$rpl-theme-menu-horizontal-hover-color: var(--rpl-theme-menu-horizontal-hover-color, rpl_color('secondary')) !default;
+$rpl-theme-section-menu-item-link-active-background-color: var(--rpl-theme-section-menu-item-link-active-background-color, rpl_color('primary')) !default;
+$rpl-theme-section-menu-active-left-bar: var(--rpl-theme-section-menu-active-left-bar, url('data:image/svg+xml,%3Csvg%20width%3D%224%22%20height%3D%221%22%20viewBox%3D%220%200%204%201%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Crect%20width%3D%224%22%20height%3D%221%22%20fill%3D%22%23#{str-slice(quote(rpl_color("secondary")), 2)}%22%2F%3E%3C%2Fsvg%3E')) !default;
+
+// Hero banner
+$rpl-theme-hero-banner-title-color: var(--rpl-theme-hero-banner-title-color, rpl_color('primary')) !default;
+$rpl-theme-hero-banner-title-background-color: var(--rpl-theme-hero-banner-title-background-color, rpl_color('white')) !default;
+
+// Footer
+$rpl-theme-site-footer-background-color: var(--rpl-theme-site-footer-background-color, rpl-color('dark_primary')) !default
+$rpl-theme-site-footer-border-color: var(--rpl-theme-site-footer-border-color, rpl-color('primary')) !default;
+$rpl-theme-site-footer-border-color-light: var(--rpl-theme-site-footer-border-color-light, rpl-color('white')) !default;
+
+// Related links
+$rpl-theme-related-links-background-color: var(--rpl-theme-related-links-background-color, rpl_color('primary')) !default;
+
+// Site header
+$rpl-theme-site-header-logo-width: var(--rpl-theme-site-header-logo-width, 150px) !default;
+
+// Card
+$rpl-theme-card-keydates-keydate-date-background: var(--rpl-theme-card-keydates-keydate-date-background, rpl_color('secondary')) !default;
+$rpl-theme-card-keydates-keydate-date-text-color: var(--rpl-theme-card-keydates-keydate-date-text-color, rpl_color('white')) !default;
+$rpl-theme-card-navigation-featured-background-alter: var(--rpl-theme-card-navigation-featured-background-alter, rpl_color('secondary')) !default;
```

Comment:
we can move these into their own component's css? I wasn't sure so I put them here so they are all in one file but I like the idea of button.scss